### PR TITLE
Ruby/Luhn New notes

### PR DESCRIPTION
**Pull request by F3PiX**
New notes!

I'd appreciate if we use this PR for correcting errors only. There's some urgency to merge, because there are several Luhns in the queue waiting for mentoring, and thus a great test case for the Notes. 

I opened a separate issue #876 at the same time, where we can discuss the next iteration of the Luhn Notes.

### CODE REVIEW

**Comment by F3PiX** [comment]
@exercism/website-copy-ruby

File: tracks/ruby/exercises/luhn/mentoring.md [hunk 1]
```diff
@@ -1,42 +1,121 @@
...
+### General
+
+For over 50% of the solutions I have seen, the 'Instantiated Class' variant is _not_ a straightforward solution. Don't underestimate its learning potential.
+In the 'Instantiated Class' variant, almost every concept has been addressed in the preceding exercises; putting them all together, and recognizing them in the first place, turns out to be a challenge for the majority of the students. It pays off to involve those earlier exercises in your feedback.
+
+### Mentoring Strategy
+
+_First iteration: start with feedback on general design and pointing to Enumerable for the Luhn Calculation_
+* When people start with a Class and one Giant Class Method: start with just a suggestion to instantiate, without going into details. 
+* When they don't use `each_slice`, I point them to the Series exercise "for inspiration on a similar solution".  I don't give them `each_slice.
```
Comment (SleeplessByte) [inline tracks/ruby/exercises/luhn/mentoring.md]:
```suggestion
* When they don't use `each_slice`, I point them to the Series exercise "for inspiration on a similar solution".  I don't give them `each_slice`.
```

File: tracks/ruby/exercises/luhn/mentoring.md [hunk 2]
```diff
@@ -1,42 +1,118 @@
...
+Well, this is Ruby, and almost everything can be done in several ways. Make sure to acknowledge the student's own preferences (given that's idiomatic Ruby).
+* _guard clause_: variants of `valid_format?(...) && valid_luhn_sum?(...)`  
+* `chars` in the initializer, `reverse` in the luhn_sum calculation
+* `divmod(10).sum` instead of `digits.sum`; I usually save one of those as a final give-away to get rid of a ternary :-) 
+* _advanced_ Most suitable as a follow up after the instantiated class is thoroughly refactored first. Don't go here except with students who demonstrate advanced Ruby fluency. (TODO: add notes.) 
+
+```ruby
+# Class with class methods or a Module with extend self or module_function
+def self.valid?(candidate)
+  return unless valid_format?(candidate.delete(' ')
```
Comment (pgaspar) [inline tracks/ruby/exercises/luhn/mentoring.md]:
```suggestion
  return unless valid_format?(candidate.delete(' '))
```

File: tracks/ruby/exercises/luhn/mentoring.md [hunk 3]
```diff
@@ -1,42 +1,118 @@
...
+  candidate.delete(' ')
+    .to_i
+    .digits
+    .yield_self(&method(:luhn_sum)
```
Comment (pgaspar) [inline tracks/ruby/exercises/luhn/mentoring.md]:
```suggestion
    .yield_self(&method(:luhn_sum))
```

**Comment by F3PiX** [comment]
@pgaspar Made the changes, thnx
When you approve, can you merge too please?

**Comment by F3PiX** [comment]
🎉

**Pull request merged**